res/graphics/royal cypher graphics/Postbox-Royal-Cypher-EIIR.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="128"
+   height="128"
+   id="svg2"
+   version="1.1"
+   inkscape:version="1.0.1 (3bc2e813f5, 2020-09-07, custom)"
+   sodipodi:docname="Postbox-Royal-Cypher-EIIR.svg">
+  <defs
+     id="defs4" />
+  <sodipodi:namedview
```

Comment:
You might want to open the SVGs in Inkscape again, and save them as "Optimized SVG". This removes a lot of the clutter in the SVG code.

Reply:
That's what is done with all SVG files in the repository. But don't worry about it, I can do that after it is merged too